### ISSUE TITLE
unable to trigger "swipe up to open tab switcher"

### ISSUE THREAD

**Issue by TsXor** [kiwibrowser/src.next#295]
It's hard to trigger when this area conflicts with the 3 buttons below.
![Screenshot_20210808_195651](https://user-images.githubusercontent.com/74352334/128631355-1c092bc7-1c40-426e-a329-2a9d9885491f.jpg)

please expand that area to this size:
![Screenshot_20210808_195728](https://user-images.githubusercontent.com/74352334/128631370-935acee3-d957-4e91-94d8-f653185f450d.jpg)

**Comment by TsXor** [comment]
Decription this in detail:
To trigger "swipe up to open tab switcher", I need to place my finger on a specific area and swipe up. I did an experiment and found that the area mentioned is what I marked in picture 1.
It's VERY hard to trigger this action on my phone, or probably lots of other phones since this area will also trigger the 3 buttons below. If you switch to gesture control, it will become IMPOSSIBLE to trigger because swipe up from the bottom will enter recent tasks.
Expand this area to what I marked in picture 2 will help.
